includes/class-wc-gateway-ppec-checkout-handler.php

```diff
@@ -60,7 +60,7 @@ public function __construct() {
 	 */
 	public function init() {
 		if ( version_compare( WC_VERSION, '3.3', '<' ) ) {
-			add_filter( 'wc_checkout_params', array( $this, 'filter_wc_checkout_params' ), 10, 1 );
+			add_filter( 'wc_checkout_params', array( $this, 'filter_wc_checkout_params' ), 10, 2 );
 		} else {
 			add_filter( 'woocommerce_get_script_data', array( $this, 'filter_wc_checkout_params_post33' ), 10, 2 );
```

Comment:
@bor0 this filter still references the remove function.

Reply:
Ah good catch man! Thanks! Updated in 41401dd.